nemo_skills/pipeline/utils/exp.py

```diff
@@ -433,9 +433,13 @@ def add_task(
     executors = []
     # assuming server always has the largest resources request, so it needs to go first
     if server_config is not None and server_config["num_gpus"] > 0:
-        server_cmd, num_server_tasks = get_server_command(**server_config, cluster_config=cluster_config)
-        if "container" not in server_config:
-            server_container = cluster_config["containers"][server_config["server_type"]]
+        # do not pass container into the command builder
+        _server_cfg = {k: v for k, v in server_config.items() if k != "container"}
```

Comment:
would it be a bit simpler and more explicit to do
```suggestion
        server_container = server_config.pop('container', cluster_config["containers"][server_config["server_type"]])
```
and then remove the assignment below?

Reply:
Good idea.